CONTRIBUTING.md

```diff
@@ -33,7 +33,7 @@ Now you can build and run BotKube by one of the following ways
    ```sh
    $ make build
    $ make container-image
-   $ docker tag infracloudio/botkube:latest <your_account>/botkube:latest
+   $ docker tag infracloudio/botkube:latest-amd64 <your_account>/botkube:latest
```

Comment:
So for local building goreleaser would build all platform images ? can we restrict it to specific platforms ? also manage image tag internally.

Reply:
Yes, goreleaser will build all the images. Image packaging does not take much time. If we don't want to use gorelease, we'll have to maintain a separate Dockerfile for a platform-specific build.